### DISCUSSION TITLE
posts/100-days-of-homelab-1-year-later/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#263] [Comments]
# posts/100-days-of-homelab-1-year-later/

A year ago I started a challenge that encouraged everyone to join the #100DaysOfHomeLab challenge, a challenge designed to help improve your skills in IT. This is similar to any of the “100 Days” challenges - pick a topic, stick with it for 100 days, and form a habit. Some of you might be asking what a “HomeLab” is, and I think in it’s simplest terms it’s a “lab environment” mostly at home. Think of this as a test environment to learn about technology without the fear of breaking anything. If you’d like to learn more about HomeLabs and how to get started, I summarized it in a video.

https://technotim.live/posts/100-days-of-homelab-1-year-later/

**Comment by 0xRyuzak1** [comment]
Excuse me can you please tell me how could you be able to make [giscus[bot] to automatically create a Discussion when you create new post without waiting for the first comment to create the Discussion as an eg. this Discussion was created even before i make this comment

**Reply by 0xRyuzak1** [reply]
@timothystewart6

**Reply by timothystewart6** [reply]
Hey, not sure, I am just using the giscuss plugin!
